### ISSUE TITLE
Showing Historical Data

### ISSUE THREAD

**Issue by valschmidt** [foxglove/community#256]

**Comment by jtbandes** [comment]
To clarify, is this mostly about the Plot panel?

This could be a small adjustment to the settings provided by the Plot panel, or it could be satisfied by a larger project: https://github.com/foxglove/community/issues/195

**Comment by valschmidt** [comment]
The plot panel, yes. 
I'm not sure of the pros/cons of doing this as part of pause/rewind functionality in foxglove/community#195. It seems to me, it is not quite the same, in the sense that one needs to change the horizontal scale of the plot to see a longer time history all at once, rather than just rewind it.

**Comment by jtbandes** [comment]
Yes, I suppose even if we implement foxglove/community#195, we would still need an x range setting on the plot panel to easily control which part of the range is shown, even if the amount of available data is controlled by the app-level "DVR" setting.
